.travis.yml

```diff
@@ -70,37 +70,79 @@ jobs:
         - if [[ "$git_status_out" == *"docs/README.md"* ]]; then echo "Error- Components version was not changed. Download pre-commit tool";exit 1; fi
 
     - stage: Roles tests
-      name: Search Role Verification
+      name: Search tests on Centos
+      env:
+        - "MOLECULE_ROLE_IMAGE=centos:7"
       script:
-        - molecule --version
-        - cd roles/search/
-        - molecule converge
-        - sleep 1m
-        - molecule verify
-        - molecule destroy
+        - cd roles/search && molecule test
 
     - stage: Roles tests
-      name: Common Role Verification
+      name: Search tests on Ubuntu
+      env:
+        - "MOLECULE_ROLE_IMAGE=ubuntu:20.04"
+      script:
+        - cd roles/search && molecule test
+
+    - stage: Roles tests
+      name: Common tests on Centos
+      env:
+        - "MOLECULE_ROLE_IMAGE=centos:7"
       script:
         - cd roles/common/ && molecule test
 
     - stage: Roles tests
-      name: Java Role Verification
+      name: Common tests on Ubuntu
+      env:
+        - "MOLECULE_ROLE_IMAGE=ubuntu:20.04"
+      script:
+        - cd roles/common/ && molecule test
+
+    - stage: Roles tests
+      name: Java tests on Centos
+      env:
+        - "MOLECULE_ROLE_IMAGE=centos:7"
+      script:
+        - cd roles/java/ && molecule test
+
+    - stage: Roles tests
+      name: Java tests on Ubuntu
+      env:
+        - "MOLECULE_ROLE_IMAGE=ubuntu:20.04"
       script:
         - cd roles/java/ && molecule test
 
     - stage: Roles tests
-      name: Tomcat Role Verification
```

Comment:
Given the repo role is dependant on the tomcat role and we doubled the number of tests I think we can remove testing this specific role